src/js/plugins/plugin.autoplay.js

```diff
@@ -0,0 +1,130 @@
+/*global BookReader */
+
+/**
+ * Plugin which adds an autoplay feature. Useful for kiosk situations.
+ */
+jQuery.extend(BookReader.defaultOptions, {
+  enableAutoPlayPlugin: true
+});
+
+/**
+ * @augments BookReader.setup
```

Comment:
I think `@override` is the best for these; `augments` looks like it's only for `class`es/inheritance. (ditto elsewhere)

```suggestion
 * @override
```